learntools/machine_learning/nbs/tut7.ipynb

```diff
@@ -0,0 +1,154 @@
+    "_cell_guid": "677b20af-bc56-4db1-aac3-1010389367ba",
+    "_uuid": "c5294634157ca8c52d35c999ee949023b9544ee1"
+   },
+   "source": [
+    "In addition to your training data, there will be test data. This is frequently stored in a file with the title `test.csv`. This data won't include a column with your target (y), because that is what we'll have to predict and submit.  Here is sample code to do that. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "_cell_guid": "a023518d-78e0-4e58-a243-6d1fe98992b9",
+    "_uuid": "7bc843ef333847d305f576489d3e2aa3afc9384d",
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Read the test data\n",
+    "test = pd.read_csv('../input/test.csv')\n",
+    "# Treat the test data in the same way as training data. In this case, pull same columns.\n",
+    "test_X = test[predictor_cols]\n",
+    "# Use the model to make predictions\n",
+    "predicted_prices = my_model.predict(test_X)\n",
+    "# We will look at the predicted prices to ensure we have something sensible.\n",
+    "print(predicted_prices)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "258e450a-d82c-4a7f-af22-cc695f74b4e3",
+    "_uuid": "49e253dc359af831d8e248979b90c70dfcbef99a"
+   },
+   "source": [
+    "# Prepare Submission File\n",
+    "We make submissions in CSV files.  Your submissions usually have two columns: an ID column and a prediction column.  The ID field comes from the test data (keeping whatever name the ID field had in that data, which for the housing data is the string 'Id'). The prediction column will use the name of the target field.\n",
+    "\n",
+    "We will create a DataFrame with this data, and then use the dataframe's `to_csv` method to write our submission file.  Explicitly include the argument `index=False` to prevent pandas from adding another column in our csv file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "_cell_guid": "092f5aed-0de4-4295-80ee-c193c8b84edd",
+    "_uuid": "da5946c5fbc32fdb537788cd6ad5da1d6d698ae6",
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "my_submission = pd.DataFrame({'Id': test.Id, 'SalePrice': predicted_prices})\n",
+    "# you could use any filename. We choose submission here\n",
+    "my_submission.to_csv('submission.csv', index=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "ab2280a4-9c65-4810-8813-831f4dcd86e1",
+    "_uuid": "97f80b7f76066fd78cdb5137757fb3c63abd0519"
+   },
+   "source": [
+    "# Make Submission\n",
+    "Hit the blue **Commit and Run** button at the top of your notebook screen.  It will take some time for your kernel to run.  When it has finished your navigation bar at the top of the screen will have a tab for **Output**.  This only shows up if you have written an output file (like we did in the **Prepare Submission File** step).  \n",
```

Comment:
Reading these instructions makes me wonder: was I supposed to fork this notebook at the beginning?